content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,326 @@
+=================
+Create quotations
+=================
+
+In Odoo **Sales**, quotations can be created and sent to customers. Once a quotation has been
+confirmed, it officially turns into a *sales order*, which can then be invoiced and paid for.
+
+.. _sales/quotation-settings:
+
+Quotation settings
+==================
+
+To access these setting options, navigate to :menuselection:`Sales app --> Configuration -->
+Settings`, and scroll to the :guilabel:`Quotations & Orders` section.
+
+.. image:: create_quotations/quotations-orders-section.png
+   :align: center
+   :alt: The Quotations and Orders section on the Odoo Sales app Settings page.
+
+- :guilabel:`Online Signature`: Request an online signature to confirm orders.
+- :guilabel:`Quotation Templates`: Enable this option to create quotation templates featuring
+  standard product offers, which are then selectable on quotation forms. When this checkbox is
+  ticked, an additional field, :guilabel:`Default Template`, appears, along with a link to the
+  :guilabel:`Quotation Templates` page.
+- :guilabel:`Default Quotation Validity`: Determine a set amount (in :guilabel:`days`) that
+  quotations can remain valid for.
+- :guilabel:`Lock Confirmed Sales`: Ensure no further edits can be made to confirmed orders.
+- :guilabel:`Online Payment`: Request an online prepayment from customers to confirm orders. Request
+  a full or partial payment (via down payment). When this checkbox is ticked, an additional field,
+  :guilabel:`Prepayment amount (%)`, appears. There is also a link to the :guilabel:`Payment
+  Providers` page.
+- :guilabel:`Quotation Builder`: Enable this option to utilize Odoo's building block options and
+  style elements to design quotation templates.
+- :guilabel:`Sale Warnings`: Get warning messages about orders that include specific products or
+  customers.
+- :guilabel:`Pro-Forma Invoice`: Send pro-forma invoices to customers.
+
+To activate any of these settings, tick the checkbox beside the desired option(s). Then, click
+:guilabel:`Save`.
+
+Quotations dashboard
+====================
+
+The *Quotations* dashboard is the page that appears when the :menuselection:`Sales app` is opened.
```

Comment:
I might be good to say how you can access the dashboard from elsewhere in the app.

Reply:
I had that in the v17 version of this doc, but was instructed to remove it during the review process, so I chose to keep that element out of this v16 version. But, if I get a similar note down the line, I'll be sure to add it in. No problem.